examples/example/App.tsx

```diff
@@ -92,28 +97,9 @@ const theme = makeTheme({
     wide: {},
     narrow: {},
   },
-  // space: [0, 4, 8, 16, 32, 64, 128, 256, 512],
```

Comment:
i removed stuff that looked redundant and added some `console.logs` and things to silence linter warnings so that typescript errors were easier to spot.